nitrates/analysis_seeds/UpperLimits.py

```diff
@@ -1,28 +1,56 @@
 import numpy as np
 from astropy.io import fits 
-from astropy.table import Table, vstack
-from astropy.wcs import WCS
+from astropy.table import Table
 import os
 import time
 import matplotlib.pyplot as plt
 from matplotlib import cm
 import pandas as pd
-from copy import copy, deepcopy
 import logging, traceback
 import sys
 import argparse
 import time
+import numpy
+from datetime import datetime, timedelta
+from astropy.time import Time
+import json
+
 
 from ..lib.event2dpi_funcs import det2dpis, mask_detxy
 from ..lib.sqlite_funcs import get_conn
 from ..lib.dbread_funcs import get_info_tab
 from ..lib.coord_conv_funcs import  convert_theta_phi2radec
 
-from ..analysis_seeds.do_full_rates import *
+from .do_full_rates import *
 
 from ..lib.calc_BAT_ul import *
 from ..config import resp_dname
 
+from gbm.data import HealPix
```

Comment:
why do you need these gbm healpix?

Reply:
that is used to create a map from a polygon, if given as input

Reply:
ok move all this stuff to UtilityBelt